baybe/recommenders/pure/base.py

```diff
@@ -31,6 +31,22 @@ class PureRecommender(ABC, RecommenderProtocol):
     """Allow to make recommendations that were measured previously.
     This only has an influence in discrete search spaces."""
 
+    user_callable: Optional[
+        Callable[
+            [
+                "PureRecommender",
+                Optional[SearchSpace],
```

Comment:
to have it match the signature its being called with, the first 2 `Optional`s should be removed